app/views/inthenewspoets/items/show.html.erb

```diff
@@ -19,9 +19,12 @@
   <% birthplace = @res["spatial"].select { |place| place["type"] == "birth place" } %>
   <% if birthplace %>
     <div>
-      Birth location: <%= birthplace[0]["country"] %>
-      <% if birthplace[0]["city"] %>
-        , <%= birthplace[0]["city"]  %>
+      Birth location:
+      <% if birthplace["country"].present? %>
+        <%= birthplace["country"] %>
+      <% end %>
+      <% if birthplace["city"].present? %>
+        , <%= birthplace["city"]  %>
```

Comment:
This commit is on two PRs which should be avoided. Either rebase and drop this commit or add another commit that makes it match exactly how the code ends up in the other PR: https://github.com/CDRH/african_poetics/pull/271

Reply:
done